packages/main/src/SegmentedButton.js

```diff
@@ -210,6 +212,11 @@ class SegmentedButton extends UI5Element {
 		return this;
 	}
 
+	_onmouseup(event) {
```

Comment:
Currently we could trigger mousedown on an SegmentedButtonItem and finish with a mouse up over another. 
This would result into the last SegmentedButtonItem receiving focus and selection without a mousedown over it.
In order to be compliant with how other buttons behave we shouldn't trigger selection and focus on mouse up,
but rather trigger focus on mouse down and trigger selection only of mousedown and mouseup are executed over a single element.